components/brave_search_conversion/pref_names.h

```diff
@@ -27,6 +27,11 @@ inline constexpr char kLatestDDGBannerTypeFirstShownTime[] =
 inline constexpr char kShowNTPSearchBox[] =
     "brave.brave_search.show-ntp-search";
 
+// Determines whether the search box on the NTP prompts the user to enable
+// search suggestions.
+inline constexpr char kPromptEnableSuggestions[] =
+    "brave.brave_search.ntp-search-prompt-enable-suggestions";
```

Comment:
nit: using under bar(`_`) is our convention instead of hyphen(`-`)?

Reply:
Nice catch!